### ISSUE TITLE
Export & Import of blocks into another website doesn't update media IDs in block code

### ISSUE THREAD

**Issue by boomtemp123** [WordPress/gutenberg#19968]
**Describe the bug**
If you export via the WordPress export tool or by using the "Copy All Content" option from Gutenberg the post_content imported/pasted into a new website contains references to the old website's media IDs which do not exist on the new website. These IDs could be used for another media object in future.

**To reproduce**
1. Create a page with some image/media based blocks
2. Export a page using WordPress standard exporter or copy all content by clicking "Copy All Content" option from Gutenberg's right menu.
3. Import/paste into a new website
4. View the code in Gutenberg's code editor
5. You will see references to media IDs which do not exist on the new website, these are the old media IDs from the old website.

**Expected behavior**
Ideal is some way of the media items being transferred to the new website, if not just don't use the old IDs as there could be clashes in future.

**Comment by boomtemp123** [comment]
Note that this issue has the [feature] reusable block label - but it effects all blocks not just reusable.

**Comment by noisysocks** [comment]
Yes that's true. All blocks that use an "id" attribute have this issue: images, galleries, reusable blocks, etc.
